service/share/discovery/discoverer.go

```diff
@@ -0,0 +1,75 @@
+package discovery
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/libp2p/go-libp2p-core/discovery"
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+)
+
+const (
+	// PeersLimit is max amount of peers that will be discovered
+	PeersLimit = 5
+	// peerWeight total weight of discovered peers
```

Comment:
```suggestion

	// peerWeight total weight of discovered peers
```

we should separate exported vars from unexported